app/services/work_packages/bulk/copy_service.rb

```diff
@@ -32,11 +32,48 @@ class CopyService < BulkedService
       def initialize(user:, work_packages:)
         super
 
+        @wp_map = {}
```

Comment:
Just for convenience we can declare an `attr_reader :wp_map` for this variable. Then later on it can be referenced without the `@`. ie: `wp_map.store(work_package.id, wp_copy.result.id)`